source/iostreams.tex

```diff
@@ -15762,7 +15763,7 @@
 \pnum
 \begin{note}
 The current path as returned by many operating systems is a dangerous
-  global variable. It might be changed unexpectedly by third-party or system
+  global variable. It can be changed unexpectedly by third-party or system
```

Comment:
The second sentence isn't true in isolation -- we don't know that it can be changed asynchronously. How about this:
```suggestion
  global variable, and can be changed unexpectedly by third-party or system
```

Reply:
fixed